server/routes/mail.coffee

```diff
@@ -692,7 +692,7 @@ sendNextStepsEmail = (user, now, daysAgo) ->
     return
 
   LevelSession.find({creator: user.get('_id') + ''}).select('levelName levelID changed state.complete playtime').lean().exec (err, sessions) ->
-    return log.error "Couldn't find sessions for #{user.get('email')}: #{err}" if err
+    return log.error "Couldn't find sessions for #{user.get('email')} #{user.get('name')}: #{err}" if err
```

Comment:
Probably should have these put in the user id instead. That's simpler and not subject to change, and this is for debugging anyway.